anchore_engine/db/entities/policy_engine.py

```diff
@@ -75,8 +75,8 @@ class FeedMetadata(Base, UtilMixin):
     enabled = Column(Boolean, default=True)
 
     @classmethod
-    def get_by_name(cls, name):
-        return FeedMetadata.query.filter(name=name).scalar()
+    def get_by_name(cls, session, name):
+        return session.query(cls).filter(cls.name == name).scalar()
```

Comment:
I did that because its a class variable and the class is available in the function but the code was referencing the class by name as opposed to using the cls reference that was available. Should behave the same as what it was before, but I can change it back if you want

Reply:
That would make it a little clearer. SQLAlchemy classes are special in that sense that they carry a lot of context. Reading this code it's not clear to which context you are querying